frontend/elements/src/entry/jig/publish/page-2.ts

```diff
@@ -1,12 +1,12 @@
-import { MEDIA_UI } from '@utils/path';
+import { mediaUi } from '@utils/path';
 import { LitElement, html, css, customElement, property } from 'lit-element';
+import "@elements/entry/home/TOSORT/placeholder"; 
```

Comment:
jig shouldn't be importing elements from home, if it's common to multiple entries, should probably be in `core` somewhere